cmd/mavenBuild.go

```diff
@@ -60,6 +64,7 @@ func runMavenBuild(config *mavenBuildOptions, telemetryData *telemetry.CustomDat
 			"-DincludeTestScope=false",
 			"-DincludeLicenseText=false",
 			"-DoutputFormat=xml",
+			"-DoutputName=" + mvnBomFilename,
```

Comment:
not sure what happens here for multi-module maven builds ...
@chrsch-dev what do you think?

Reply:
Hm, since the default name would just be `bom` I don't think this change would raise any new problem.